### PR TITLE
[DOCS] Clarify that v2.0+ hyphenation files aren't supported

### PR DESCRIPTION
**Pull request by jgkirschbaum**
- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md)?
- If submitting code, have you built your formula locally prior to submission with `gradle check`?
- If submitting code, is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.
- If submitting code, have you checked that your submission is for an [OS and architecture that we support](https://www.elastic.co/support/matrix#show_os)?
- If you are submitting this code for a class then read our [policy](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md#contributing-as-part-of-a-class) for that.

### CODE REVIEW

**Comment by jgkirschbaum** [comment]
CLA now signed.

File: docs/reference/analysis/tokenfilters/hyphenation-decompounder-tokenfilter.asciidoc [hunk 1]
```diff
@@ -60,7 +60,7 @@ compatible files are supported.
 For example FOP XML hyphenation pattern files, refer to:
 
 * http://offo.sourceforge.net/#FOP+XML+Hyphenation+Patterns[Objects For Formatting Objects (OFFO) Sourceforge project]
-* https://sourceforge.net/projects/offo/files/offo-hyphenation/1.2/offo-hyphenation_v1.2.zip/download[offo-hyphenation_v1.2.zip direct download]
+* https://sourceforge.net/projects/offo/files/offo-hyphenation/1.2/offo-hyphenation_v1.2.zip/download[offo-hyphenation_v1.2.zip direct download] (v2.0 and above hypheniation pattern files are not supported)
```
Comment (dliappis) [inline docs/reference/analysis/tokenfilters/hyphenation-decompounder-tokenfilter.asciidoc:63]: typo: `hypheniation` -> `hyphenation`

**Review by dliappis — DISMISSED** [review]
Thank you for the PR. I left a comment for a typo.

File: docs/reference/analysis/tokenfilters/hyphenation-decompounder-tokenfilter.asciidoc (hunk 1 above)
Comment (jrodewig) [inline docs/reference/analysis/tokenfilters/hyphenation-decompounder-tokenfilter.asciidoc:63]:
```suggestion
* https://sourceforge.net/projects/offo/files/offo-hyphenation/1.2/offo-hyphenation_v1.2.zip/download[offo-hyphenation_v1.2.zip direct download] (v2.0 and above hyphenation pattern files are not supported)
```

**Review by jrodewig — APPROVED** [review]
LGTM. Thanks @jgkirschbaum

**Comment by jrodewig** [comment]
@elasticmachine test this please

**Pull request merged**